conf/messages.en

```diff
@@ -1298,6 +1298,16 @@ pdf.template.title = Export accompanying document (EAD)
 pdf.template.mrn = MRN
 pdf.template.eori = Declarant EORI number
 pdf.template.declarationType = Declaration type
+pdf.template.declarationType.EXA = Full Exports Declaration at Frontier
+pdf.template.declarationType.EXB = Simplified Occasional Exports Declaration at Frontier
+pdf.template.declarationType.EXC =Simplified Exports Declaration at Frontier
```

Comment:
Petty comment, but just to be consistent, it would be nice to provide a space after the "equal" sign